README.md

```diff
@@ -97,14 +95,23 @@ Syntax highlighting recognizes all versions of Puppet and displays as you type.
 - Puppet Grammar
 - Module metadata files
 
+<!-- ![syntax](https://raw.githubusercontent.com/lingua-pupuli/puppet-vscode/master/docs/assets/syntax_highlighting.png) -->
+![syntax](docs/assets/syntax_highlighting.png)
+
 ### Auto Completion
 
 Auto complete Puppet types and classes and their parameter sets as you type. Tab completion works as you would expect.
 
+<!-- ![autocomplete](https://raw.githubusercontent.com/lingua-pupuli/puppet-vscode/master/docs/assets/auto_complete.gif) -->
+![autocomplete](docs/assets/auto_complete.gif)
+
 ### Hover Support
 
 Hovering over any resource, class declaration or other Puppet symbol provides instant contextual information. For example, hovering over the resource declaration lists the type name and parameter list, with all relevant help text.
 
+<!-- ![hover](https://raw.githubusercontent.com/lingua-pupuli/puppet-vscode/master/docs/assets/hover_support.gif) -->
+![hover](docs/assets/hover_support.gif)
+
 ### Code Snippets
 
 As part of IntelliSense and Snippets, you can quickly create templates to automate repetitive series of code.
```

Comment:
This implies I can create my own snippets, but there's no instructions or examples - is this just using the VSCode features or an actual feature of the extension?